### ISSUE TITLE
validate() doesn't check the "Not Before" date

### ISSUE THREAD

**Issue by navzam** [mreinstein/alexa-verifier#23]
In validate-cert.js, this code
```node
var notAfter = new Date(cert.validity.notAfter)
var remainingDays = notAfter.getTime() - new Date().getTime()
// check that the signing certificate has not expired (examine both the Not
// Before and Not After dates)
if (remainingDays < 1) {
  return 'certificate expiration check failed'
}
```
checks the "Not After" date, but it doesn't check the "Not Before" date.

**Comment by mreinstein** [comment]
This is another case of the comment not matching the actual check.  I'd be open to either updating the comment, or accepting a PR that adds the "not before" check.

good catch! 👍

**Comment by navzam** [comment]
I'm in favor for adding a "not before" check, since the [Checking the Signature of the Request](https://developer.amazon.com/public/solutions/alexa/alexa-skills-kit/docs/developing-an-alexa-skill-as-a-web-service#checking-the-signature-of-the-request) docs say to check both `Not After` and `Not Before`

**Comment by mreinstein** [comment]
closed via #25
